src/common/zipkin_trace.h

```diff
@@ -3,8 +3,6 @@
 #ifndef COMMON_ZIPKIN_TRACE_H
 #define COMMON_ZIPKIN_TRACE_H
 
-#include <string>
-#include "include/encoding.h"
```

Comment:
This was needed to avoid a compile error since ```::encode``` and ```::decode``` are used in this header w/o the declarations